src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -32,7 +29,16 @@ export default class SidebarMenu extends React.Component {
     }
   }
   render() {
-    let self = this
+    let self = this;
+    function includes(array, value) {
```

Comment:
what was the problem with lodash?

Reply:
the standard lodash.include method is not suitable for the new sidebar.json structure

Reply:
the standard lodash.include is not suitable for the new sidebar.json structure, since now each files element contains the full structure {name, folder, indexFile, files}.
lodash.include will look for a string with the file name in the root of the files array